vaadin-combo-box-flow-parent/vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/StringItemsWithTextRendererIT.java

```diff
@@ -36,14 +37,13 @@ public void stringItemsAreRendered() {
 
         $("vaadin-combo-box").id("list").sendKeys(Keys.ARROW_DOWN);
 
-        WebElement overlay = findElement(
-                By.tagName("vaadin-combo-box-overlay"));
-        WebElement content = getInShadowRoot(overlay, By.id("content"));
-        WebElement selector = getInShadowRoot(content, By.id("selector"));
+        TestBenchElement overlay = $("vaadin-combo-box-overlay").first();
+        TestBenchElement content = overlay.$("*").id("content");
+        TestBenchElement selector = content.$("*").id("selector");
         List<String> items = selector
                 .findElements(By.tagName("vaadin-combo-box-item")).stream()
-                .map(item -> getInShadowRoot(item,
-                        By.cssSelector("flow-component-renderer")))
+                .map(item -> item
+                        .findElement(By.cssSelector("flow-component-renderer")))
```

Comment:
```suggestion
                .map(item -> item.$("*")
                        .$("flow-component-renderer"))
```

Reply:
looks i have fixed it in the other way.. let us see the validation